hr_hr.lang

```diff
@@ -1,215 +1,349 @@
+gui.xaero_howto_save=Spremi promjene klikom na gumb "Potvrdi".
+gui.xaero_howto_cancel=Poništi promjene klikom na gumb "Poništi".
+gui.xaero_minimap_settings=Postavke mini karte
+gui.xaero_miscellaneous_settings=Razne postavke
+gui.xaero_notification_settings=Postavke obavijesti
+gui.xaero_better_pvp_settings=Better PVP postavke
+gui.xaero_server_disabled=Poslužitelj je možda onemogućio neke mogućnosti moda.
+gui.xaero_waypoints=Putne točke
+gui.xaero_box_potion_effects=Sučelje koje prikazuje trenutno aktivne efekte napitaka.
+gui.xaero_box_armour_status=Sučelje koje prikazuje oklop koji trenutno nosiš i predmet u ruci.
+gui.xaero_box_sprint=Trčanje držanjem samo jedne tipke (podesivo).
+gui.xaero_box_sneak=Omogućuje uključivanje/isključivanje šuljanja i stajanje šuljajući se bez potrebe držanja tipke.
+gui.xaero_box_notifications=Obavijesti na zaslonu koje prikazuju različite stvari kao što su: niska glad, nisko zdravlje, eksplozija u blizini ili kad si pogođen strijelom/ama.
+gui.xaero_box_xp=Bodovi iskustva koje si dobio ili izgubio lete preko tvog zaslona.
+gui.xaero_box_quick_use=Sistem tipki sličan akcijskoj traci koji ti dopušta automatsko korištenje određenog predmeta u svojoj vrućoj traci bez potrebe za pomicanjem sa svojeg trenutno držanog predmeta. Pritiskanje određene tipke omogućuje ti da nastaviš s korištenjem predmeta kao da si pritiskao desni gumb na mišu. Npr.: postavljanje baklji, TNT-a, bacanje napitaka, jedenje hrane, pijenje.
+gui.xaero_box_minimap=Mini karta prikazuje ti tvoje najbliže okruženje i entitete.
+gui.xaero_centered=Centrirano:
+gui.xaero_press_c=(pr. C)
+gui.xaero_flipped=Okrenuto:
+gui.xaero_press_f=(pr. F)
+gui.xaero_preset_fancy=Šminkerski
+gui.xaero_preset_classic=Klasično
+gui.xaero_preset_social=Socijalno
+gui.xaero_potion_status=Stanja/e napit(a)ka
+gui.xaero_archery_status=Prikaži strelice ulijevo
+gui.xaero_sprint=Bolje trčanje
+gui.xaero_sneak=Prebaci šuljanje
+gui.xaero_enchants_color=Boje čarolija
+gui.xaero_durability=Trajnost u
+gui.xaero_hunger_setting=Obavijest o gladi
+gui.xaero_hp_setting=Obavijest o zdravlju
+gui.xaero_explosion_setting=Obavijest o eksploziji
+gui.xaero_being_shot_setting="Pogođen si" obavijest
+gui.xaero_xp_setting=Dobiveni bodovi iskustva
+gui.xaero_custom_settings=Prilagođene postavke
+gui.xaero_reset_defaults=Vrati zadano
+gui.xaero_quick_use=Brzo korištenje predmeta
+gui.xaero_show_enchants=Prikaži čarolije
+gui.xaero_cave_maps=Špiljske karte
+gui.xaero_display_teams=Prikaži druge timove
+gui.xaero_display_waypoints=Putne točke na mapi
+gui.xaero_display_players=Prikaži igrače
+gui.xaero_display_mobs=Prikaži živa bića
+gui.xaero_display_items=Prikaži predmete
+gui.xaero_display_other=Prikaži ostalo
+gui.xaero_zoom=Zoom
+gui.xaero_minimap_size=Veličina mini karte
+gui.xaero_entity_amount=Količina entiteta
+gui.xaero_black=Crna
+gui.xaero_dark_blue=Tamno plava
+gui.xaero_dark_green=Tamno zelena
+gui.xaero_dark_aqua=Tamno tirkizna

```

Comment:
This was made by using in game translations and they sound off to me too but that's how it is. Aqua in Croatian is translated as akva but still written as aqua. Because aqua uses q and q isn't in Croatian alphabet, people do say word aqua but never write it, and instead type something close to it. Dark turquoise is closest to aqua so aqua is called dark turquoise meaning "Tamno tirkizna" as I wrote and as it stands in game. Idk why it is like that but, even though it is stupid it is how it needs to be, so I won't change it.